src/components/tool/guide/EresearchJob/EresearchJob.js

```diff
@@ -121,6 +122,60 @@ const getConfigGroups = (config, onConfigChange = () => {}) => {
         selected: (config) => config?.nodes > 0,
       };
     },
+    isArrayJob: () => {
+      const arrayJobEnabled = [
+        ["Yes", "Sets job's array attribute to True"],
+        ["No", "Sets job's array attribute to False"],
+      ];
+      return {
+        element: (key, selected) => (
+          <ConfigGroup
+            key={key}
+            title="Array Job"
+            description="Make this job an array job"
+            type="picker"
+            selected={selected}
+            inputProps={{
+              choices: arrayJobEnabled,
+              value: config?.isArrayJob,
+              onChange: onChange("isArrayJob"),
+            }}
+            showAlert={config?.isArrayJob === "Yes"}
+            alertType="info"
+            alertMsg="Please make sure you understand what is an array job."
+          />
+        ),
+        show: (config) =>
+          config?.service === "Lyra" && config?.jobType === "Batch",
+        selected: (config) =>
+          isValidChoice(arrayJobEnabled, config?.isArrayJob),
+      };
+    },
```

Comment:
This should be called `jobInstances` or similar, and not default to running as an array

![image](https://github.com/user-attachments/assets/97c9efc3-1d53-4665-8981-51a6437c9acf)

And the alert should either be removed, or provide more detail.

```suggestion
    jobInstances: () => {
      const jobInstances = [
        ["Standalone", "Run a single instance of the job"],
        ["Array", "Run many instances of the job in parallel"],
      ];
      return {
        element: (key, selected) => (
          <ConfigGroup
            key={key}
            title="Job Instances"
            description="Run a single instance, or many instances of the job."
            type="picker"
            selected={selected}
            inputProps={{
              choices: jobInstances,
              value: config?.jobInstances,
              onChange: onChange("jobInstances"),
            }}
          />
        ),
        show: (config) =>
          config?.service === "Lyra" && config?.jobType === "Batch",
        selected: (config) => isValidChoice(jobInstances, config?.jobInstances),
      };
    }
```

Reply:
Updated

Reply:
The help text could be more descriptive as in my suggestion, eg:

```
      const jobInstances = [
        ["Standalone", "Run a single instance of the job"],
        ["Array", "Run many instances of the job in parallel"],
      ];
```

Also you updated the wrong description label

Reply:
Updated